src/styles/style-helpers.ts

```diff
@@ -25,6 +25,16 @@ function isTheme(args: ThemeArgs): args is Theme {
 export const getTheme = (args: ThemeArgs): Theme =>
   isTheme(args) ? args : args.theme;
 
+type StyleFn =
+  | ((theme: Theme) => SerializedStyles)
+  | ((args: ThemeArgs) => SerializedStyles)
+  | false
+  | null
+  | undefined;
+
+export const cx = (...styleFns: StyleFn[]) => (theme: Theme) =>
+  styleFns.map((styleFn) => styleFn && styleFn(theme));
```

Comment:
We could also use something like lodash's `isFunction` here, what do you think?

Reply:
I'm slowly trying to remove Circuit UI's dependency on lodash and use vanilla JS instead. The note about checking for a function is good though, at least during development. I have the same concern here as on the spacing mixin PR: this is a hot code path that needs to be optimized for performance. 

If a developer passes anything but a function or a falsy value, the function shouldn't gracefully handle that. It should throw a visible error since this is likely an oversight/bug that should be fixed.

Reply:
Makes sense!